test/tokenizedExits.js

```diff
@@ -0,0 +1,152 @@
+import chai from 'chai';
+
+import { Period, Block, Tx, Input, Output, Outpoint } from 'parsec-lib';
+
+const ParsecBridge = artifacts.require('./ParsecBridge.sol');
+const PriorityQueue = artifacts.require('./PriorityQueue.sol');
+const ExitToken = artifacts.require('./ExitToken.sol');
+const TxLib = artifacts.require('./TxLib.sol');
+const SimpleToken = artifacts.require('SimpleToken');
+
+chai.use(require('chai-as-promised')).should();
+
+contract('TokenizedExits', (accounts) => {
+
+  const alice = accounts[0];
+  const alicePriv = '0x278a5de700e29faae8e40e366ec5012b5ec63d36ec77e8a2417154cc1d25383f';
+  const p = [];
+  let bridge;
+  let token;
+  let exitToken;
+  before(async () => {
+    token = await SimpleToken.new();
+
+    // deploy bridge
+    const pqLib = await PriorityQueue.new();
+    ParsecBridge.link('PriorityQueue', pqLib.address);
+    bridge = await ParsecBridge.new(4, 50, 0, 0, 50);
+    await bridge.registerToken(token.address);
+
+    exitToken = await ExitToken.new(bridge.address);
+
+    p[0] = await bridge.tipHash();
+
+    // alice auctions slot
+    await token.approve(bridge.address, 1000, {from: alice});
+    // aprove exit token contract to spend native tokens to pay the stake
+    await token.approve(exitToken.address, 1000, {from: alice});
+    await bridge.bet(0, 150, alice, alice, {from: alice}).should.be.fulfilled;
+  });
+
+  it('should set Bridge addresss correctly on ExitToken', async () => {
+    const bridgeAddress = await exitToken.bridge();
+    assert(bridgeAddress == bridge.address);
+  });
+
+  it('should allow tokenized exit of simple transfer', async () => {
+    const deposit = Tx.deposit(114, 50, alice);
+    let transfer = Tx.transfer(
+      [new Input(new Outpoint(deposit.hash(), 0))],
+      [new Output(50, exitToken.address)]
```

Comment:
why `exitToken.address` ?

Reply:
As in why send to the contract? Alice sends a tx to the contract, locking her funds, so that they can not be spent further. The funds then belong to exitToken and alice can redeem her token.

Reply:
I see. Ignore my PR then, I have to think more about it